Descent3/gamespy.cpp

```diff
@@ -124,7 +124,7 @@ int gspy_Init() {
   }
 
   // Read the config, resolve the name if needed and setup the server addresses
-  cfgpath = std::filesystem::path(Base_directory) / gspy_cfgfilename;
+  cfgpath = cf_GetWritableBaseDirectory() / gspy_cfgfilename;
```

Comment:
gamespy file can be not only in write-path

Reply:
OK. I changed that line to this:

```cpp
  cfgpath = cf_LocatePath(gspy_cfgfilename);
```